src/any_agent/tracing/agent_trace.py

```diff
@@ -318,12 +317,6 @@ def cost(self) -> CostInfo:
         sum_output_cost = 0.0
         for span in self.spans:
             if span.is_llm_call():
-                if any(
-                    key in span.attributes
-                    for key in ["gen_ai.usage.input_cost", "gen_ai.usage.output_cost"]
-                ):
-                    sum_input_cost += span.attributes.get("gen_ai.usage.input_cost", 0)
-                    sum_output_cost += span.attributes.get(
-                        "gen_ai.usage.output_cost", 0
-                    )
+                sum_input_cost += span.attributes.get(GenAI.USAGE_INPUT_COST, 0)
```

Comment:
Not sure why the if statement was removed

Reply:
I should/could have applied the changes in a separate PR but I just noticed it when applying this changes.

The `if` is irrelevant/redundant for the final behavior, as summing 0 is equivalent to not doing anything